### PR TITLE
620 product detail child prod tab

### LINKED ISSUE #620: [Product] Product/Product Family detail page - part 4 (coveo)
- [ ] Resource Tab
- [ ] Products Tab (bundle only)
- [ ] Related Products Tab

All using Coevo Atomic library

### PR DESCRIPTION
**Pull request by rgravitvl**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #620 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/us/en/products/family/spectramax-id3-id5-series-microplate-readers
- After: https://620-product-detail-child-prod-tab--danaher-ls-aem--hlxsites.hlx.page/us/en/products/family/spectramax-id3-id5-series-microplate-readers#products
- After: https://620-product-detail-child-prod-tab--danaher-ls-aem--hlxsites.hlx.page/us/en/products/family/cloneselect-imagers#products
- After: https://620-product-detail-child-prod-tab--danaher-ls-aem--hlxsites.hlx.page/us/en/products/family/m80-routine-stereo-microscopes#products

### CODE REVIEW

**Review by davenichols-DHLS — APPROVED** [review]
Nice LHS from this, I can't see the child product information on the example pages but presume we need the tabs implemented.

**Comment by duynguyen** [comment]
I think the block implementation should be independent from the tabs implementation. Without the tabs, all content is just displayed on one page. @rgravitvl any idea why the product-children block displays empty?

**Comment by rgravitvl** [comment]
> I think the block implementation should be independent from the tabs implementation. Without the tabs, all content is just displayed on one page. @rgravitvl any idea why the product-children block displays empty?

As discussed in the call, products are appearing in the page. Thanks.

File: blocks/product-children/product-children.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+export default async function decorate(block) {
+  const sku = getSKU();
+  const host = (window.location.host === 'lifesciences.danaher.com') ? window.location.host : 'stage.lifesciences.danaher.com';
+  const response = getProductResponse();
+  if (response?.length > 0 && response[0]?.raw?.objecttype === 'Family' && response[0]?.raw?.numproducts > 0) {
+    block.classList.add('pt-10');
+    block.innerHTML = childProducts;
+    await import('https://static.cloud.coveo.com/atomic/v2/atomic.esm.js');
+    await customElements.whenDefined('atomic-search-interface');
+    loadScript('/blocks/category-family/image-component.js');
```
Comment (duynguyen) [inline blocks/product-children/product-children.js:83]: let's maybe move image-component to scripts/ ?

Comment (rgravitvl) [reply blocks/product-children/product-children.js:83]: Done

**Comment by davenichols-DHLS** [comment]
@duynguyen  - are you OK to approve this one as well, now Ravi has made the changes?

**Review by duynguyen — APPROVED** [review]

**Pull request merged**